Bugfix/autocomplete small bugs (#168)
digipolisantwerp/acpaas-ui_react: CHANGELOG.md

## [Unreleased]

### Fixed

> - Autocomplete does now not crash when a non-existing defaultValue is provided

Reviewer: ```suggestion
- `Autocomplete` Fixed a crash when a non-existing defaultValue is provided
```

> - Sing Selection + Sync Mode now selects the correct value

Reviewer: I'm not sure what this means.
```suggestion
- `Autocomplete` Fixed an issue where the correct value wasn't selected in async search mode
```
